### DISCUSSION TITLE
Kibana NIDS Dashboards - No Data?

### DISCUSSION THREAD

**Discussion by arisentow** [Security-Onion-Solutions/securityonion#9040] [Unsupported Versions] [answered]
For the two "NIDS" dashboards pre-loaded with Kibana, I am not seeing any data populated. Should these work out of the box or do I need to make any configuration changes? The logs are present/searchable in Kibana. No issues with the SOC dashboards. (SO 2.3.181)

NIDS - SID Drilldown:

![image](https://user-images.githubusercontent.com/116360047/199219758-14d046ac-c26c-4132-a2c5-958fba5456cd.png)

![image](https://user-images.githubusercontent.com/116360047/199219859-c0929a8e-80fe-4edb-8ff3-e373ecedf03e.png)

z16.04:

![image](https://user-images.githubusercontent.com/116360047/199220123-606d6668-1db1-47dc-94da-5f70358b0933.png)

![image](https://user-images.githubusercontent.com/116360047/199220206-afbe9100-c7d6-4c55-929b-80b08a827d97.png)

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/kibana.html#dashboards:
> We’ve included the old 16.04 dashboards in case you performed an in-place upgrade and have any old 16.04 data. These dashboards are named with the z16.04 prefix and will only show old 16.04 data from the old *:logstash-* indices. The new Security Onion 2 dashboards are all named with the Security Onion prefix and they should be used for any new data stored in the new *:so-* indices.

**Reply by shabeebpm** [reply]
i am also getting this error, could you please explain more about this with the help of screenshot, appreciate your support

**Reply by dougburks** [reply]
@shabeebpm This is explained in my response above.

**Reply by shabeebpm** [reply]
i tried to follow the method explained here, but it didnt fix my issue, if you can support on this issue it would be appreciated

**Reply by dougburks** [reply]
@shabeebpm There is no issue to fix. The old dashboards are only for old data. Use the Kibana dashboards with the `Security Onion` prefix or use SOC dashboards:
https://docs.securityonion.net/en/2.3/dashboards.html

If you have other questions or problems, please start a new discussion rather than replying to somebody else's discussion.
